### PR TITLE
🎨 Clean up Preferences window

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR cleans up the Preferences window:

- [x] Show title together with the icon on the tab
- [x] Show General Settings page regardless the product (it used to be shown only for the billable ones). The visibility own its settings now depends on the setting:
  - [x] Credits Indicator Settings: only for billable products
  - [x] Low Disk Space Setting: exposed for all products
  - [x] Inactivity Settings: only for billable products
  - [x] Job Concurrency Settings: always hidden
  - [x] S4L User Privacy Settings: only for those products that have S4L in their catalog

S4L:
image

oSPARC:
image

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
thx!

**Review by elisabettai — APPROVED** [review]
Nice, thanks!

Small suggestion, since the first 2 tabs have very long titles ("General/Confirmation settings"), I'd propose to make the following ones slightly longer so they're (1) easy to understand, (2) they visually look nicer compared to the 2 long ones on top.
- API -> API Keys/Tokens
- Tags -> Create/Edit Tags

**Comment by odeimaiz** [comment]
> Nice, thanks!
> 
> Small suggestion, since the first 2 tabs have very long titles ("General/Confirmation settings"), I'd propose to make the following ones slightly longer so they're (1) easy to understand, (2) they visually look nicer compared to the 2 long ones on top.
> 
> * API -> API Keys/Tokens
> * Tags -> Create/Edit Tags
> 
> Obviously, if you go for this change, you'll need to update the tab titles as well

Done 👍

**Review by sanderegg — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/desktop/preferences/pages/GeneralPage.js [hunk 1]
```diff
@@ -22,16 +22,13 @@ qx.Class.define("osparc.desktop.preferences.pages.GeneralPage", {
...
-    if (preferences.getLowDiskSpaceThreshold()) {
-      this.add(this.__createLowDiskSpaceSetting());
-    }
-    this.add(this.__createInactivitySetting());
-    this.add(this.__createJobConcurrencySetting());
-    this.add(this.__createUserPrivacySettings());
+    this.__addCreditsIndicatorSettings();
+    this.__addLowDiskSpaceSetting();
+    this.__addInactivitySetting();
+    // this.__addJobConcurrencySetting();
```
[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/desktop/preferences/pages/GeneralPage.js:30]: There was a request to, for now, completely hide it.

**Pull request merged**